components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -24,7 +25,9 @@ function InputBox () {
   const isCharLimitApproaching = inputText.length >= CHAR_LIMIT_THRESHOLD
 
   const onInputChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    const parentNode = e.target.parentNode as HTMLDivElement
```

Comment:
This doesn't fire when the text is changed as a result of JS, i.e. calling `setInputText`. The obvious bug this is causing at the moment is that hitting submit doesn't reset the input size. If the message is 4 lines long, the input box remains 4 lines high until the next message starts to be typed.
How about just handling it in a `useEffect`? In fact, why do we even need to do this manually? Can't we just use react to set the data attribute value?
```jsx
<div className={styles.growWrap} data-replicated-value={inputText}>
```

Reply:
til data-* attributes are exposed directly in react dom, changed

Reply:
any attribute is